### PR TITLE
Fake transfermanager for test runs and test speed up

### PR DESCRIPTION
**Pull request by audax**
The QblJUnitTestRunner configures the BoxProvider
to use the FakeTransferManager which only writes
locally.

* Use setText instead of typeText for faster ui tests
* improve start up and test time for instrumentation tests in general
* Improve stability of many tests.

Resoves #133

### CODE REVIEW

**Comment by audax** [comment]
Apparently the emulator does not have connectivity…

**Comment by audax** [comment]
retest this

**Comment by audax** [comment]
retest this please

**Comment by audax** [comment]
It finally looks stable!

**Comment by audax** [comment]
The FileSearchUITest is not really stable, just because the RecyclerView is very quirky and throws IllegalStateExceptions like crazy.

File: qabelbox/src/main/java/de/qabel/qabelbox/activities/CreateIdentityActivity.java [hunk 1]
```diff
@@ -40,6 +41,13 @@
 public class CreateIdentityActivity extends BaseWizardActivity {
 
     public static final int REQUEST_CODE_IMPORT_IDENTITY = 1;
+    /**
+     * Fake the prefix request
+     *
+     * This is set by the QblJUnitTestRunner to prevent network requests to the block server
+     * in test runs.
+     */
+    public static boolean FAKE_COMMUNICATION = false;
```
Comment (JoGir) [inline qabelbox/src/main/java/de/qabel/qabelbox/activities/CreateIdentityActivity.java]: Hmm should not be here. We could create a FakePrefixServer and inject it or override the loadPrefixInBackground function in a TestCreateIdentityActivity

**Pull request merged**